.github/workflows/ci.yml

```diff
@@ -46,6 +46,9 @@ jobs:
     - name: Run Tests     
       run: |
         poetry run python manage.py test
+        poetry run black . --line-length=79
```

Comment:
I think we want to run `poetry run black . --line-length=79 --check` since we are just looking if the files are formatted properly